app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1114,32 +1114,42 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false
     )
 
-    var historyMetadataUIFeature by featureFlagPreference(
+    var historyMetadataUIFeature by lazyFeatureFlagPreference(
         appContext.getPreferenceKey(R.string.pref_key_history_metadata_feature),
-        default = FeatureFlags.historyMetadataUIFeature,
+        default = { appContext.components.analytics.features.homeScreen.isRecentExplorationsActive() },
         featureFlag = FeatureFlags.historyMetadataUIFeature || isHistoryMetadataEnabled
     )
 
     /**
      * Indicates if the recent tabs functionality should be visible.
      * Returns true if the [FeatureFlags.showRecentTabsFeature] and [R.string.pref_key_recent_tabs] are true.
      */
-    var showRecentTabsFeature by featureFlagPreference(
+    var showRecentTabsFeature by lazyFeatureFlagPreference(
         appContext.getPreferenceKey(R.string.pref_key_recent_tabs),
-        default = FeatureFlags.showRecentTabsFeature,
-        featureFlag = FeatureFlags.showRecentTabsFeature
+        featureFlag = FeatureFlags.showRecentTabsFeature,
+        default = { appContext.components.analytics.features.homeScreen.isRecentlyTabsActive() }
+    )
+
+    val showRecentTabsPreference by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_recent_tabs),
+        default = false
     )
 
     /**
      * Indicates if the recent saved bookmarks functionality should be visible.
      * Returns true if the [FeatureFlags.showRecentTabsFeature] and [R.string.pref_key_recent_bookmarks] are true.
      */
-    var showRecentBookmarksFeature by featureFlagPreference(
+    var showRecentBookmarksFeature by lazyFeatureFlagPreference(
         appContext.getPreferenceKey(R.string.pref_key_recent_bookmarks),
-        default = FeatureFlags.recentBookmarksFeature,
+        default = { appContext.components.analytics.features.homeScreen.isRecentlySavedActive() },
         featureFlag = FeatureFlags.recentBookmarksFeature
     )
 
+    val showRecentBookmarksPreference by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_recent_bookmarks),
+        default = false
+    )
```

Comment:
Hmm. I'm sure we should be using `showRecentBookmarksFeature` here instead of `showRecentBookmarksPreference`.

Reply:
Also: `pocketRecommendations` should also be a `lazyFeatureFlagPreference` now.